src/http/Client.zig

```diff
@@ -1040,6 +1040,16 @@ pub const Transfer = struct {
 
         try req.done_callback(req.ctx);
     }
+
+    pub fn getContentLength(self: *const Transfer) ?u32 {
+        // It's possible for this to be null even with correct code, due to
+        // request fulfillment. If transfer.fulfill is called, we won't have
+        // a handle.
+        const handle = self._handle orelse return null;
```

Comment:
It works correctly into the header callback but this function will work only if the request is not fulfill and is not done. Maybe we should a comment to specify this function is useful only during this callback.